ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPLexicalModel.swift

```diff
@@ -23,6 +23,20 @@ class KMPLexicalModel: Codable, KMPResource {
     case languages
   }
 
+  internal required init?(from resource: LanguageResource) {
+    guard let lexicalModel = resource as? InstallableLexicalModel else {
+      return nil
```

Comment:
Should passing something other than a lexical model throw an exception? I'm not all that clear on Swift's practices here.